content/applications/finance/accounting/payments/follow_up.rst

```diff
@@ -14,44 +14,61 @@ Configuration
 =============
 
 To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
-Follow-up Levels`. Several follow-up actions are available by default, and you can customize them
-any way you want or create new ones to trigger the following actions:
+Follow-up Levels`. Several follow-up actions are available by default under the
+:guilabel:`Notification`, but you can create new ones if required, and can change the **name** and
+the **number of days** after which the follow-up actions are triggered. The follow-up
+:guilabel:`Actions` available are:
 
-- Send an email;
-- :ref:`Send an SMS message <pricing/pricing_and_faq>`;
-- Print a letter;
-- :ref:`Send a letter <customer_invoices/snailmail>`;
-- Manual action (creates a task).
+- :guilabel:`Send Email`;
+- :ref:`Send a Letter <customer_invoices/snailmail>`;
+- :ref:`Send an SMS message <pricing/pricing_and_faq>`.
 
-It is also possible to automatically send a reminder by enabling the :guilabel:`Auto Execute`
-option, within a specific :guilabel:`Follow-Up Action`.
+You can use a pre-filled template for your messages by selecting a :guilabel:`Content Template`. To
+change the template used, hover over the the field and click the :guilabel:`-->`. If enabled, SMS
+messages have a specific :guilabel:`Sms Template` field.
+
+It is possible to automatically send a reminder by enabling the :guilabel:`Automatic` option, and
+attaching the *open* invoice(s) by enabling :guilabel:`Attach Invoices`, within a specific follow-up
+action.
+
+By clicking on the :guilabel:`Activity` tab, it's possible to schedule activities (tasks). That way,
+when the follow-up is triggered, an activity is automatically scheduled. To do so, enable
+:guilabel:`Schedule Activity`, and select a :guilabel:`Responsible` person for the task. Choose an
+:guilabel:`Activity Type`, and enter a :guilabel:`Summary` on how to handle the activity, if
+desired.
 
 .. tip::
    To send a reminder before the actual due date is reached, set a negative number of due days.
 
 Follow-up reports
 =================
 
-The overdue invoices you need to follow up on are available in :menuselection:`Accounting -->
-Customers --> Follow-up Reports`. From there, you  can identify all the customers that have late
-unpaid invoices.
+Overdue invoices you need to follow up on are available in :menuselection:`Accounting --> Customers
+--> Follow-up Reports`. By default, Odoo filters by :guilabel:`Customer Invoices` that are
+:guilabel:`In need of action`.
```

Comment:
Maybe to simplify, I'd talk about what's actually on the filter "overdue invoices"

![Uploading overdue filter.png…]()